### DISCUSSION TITLE
Are x-refs global?

### DISCUSSION THREAD

**Discussion by yabdab** [alpinejs/alpine#1692] [4. General]
When I try to target an x-ref inside a child component ( from a parent component )  it comes back undefined.

Is this expected?

**Comment by yabdab** [comment]
An example showing what I mean...
https://jsfiddle.net/yabdab/x4c2dvrs/17/

**Comment by ryangjchandler** [comment]
@yabdab Refs are discovered inside of the local component, i.e. accessing a ref from the child component in the parent isn't possible because it's not declared inside of the parent's scope.
